index.js

```diff
@@ -359,6 +363,80 @@
       [a, b, a],
       function(x, y) { return x; });
 
+  //# A :: (a -> b) -> a -> b
+  //.
+  //. The A combinator. Takes a function and a value and returns the result of
+  //. applying the function to the value. Equivalent to Haskell's `($)` function.
+  //.
+  //. ```javascript
+  //. > S.A(R.inc, 1)
+  //. 2
+  //.
+  //. > R.map(S.A(R.__, 100), [R.inc, Math.sqrt])
+  //. [101, 10]
+  //. ```
+  S.A =
+  def('A',
+      {},
+      [$.Function, a, b],
+      function(f, x) { return f(x); });
+
+  //# C :: (a -> b -> c) -> b -> a -> c
+  //.
+  //. The C combinator. Takes a curried binary function and two values and
+  //. returns the result of applying the function to the values in reverse.
+  //. Equivalent to Haskell's `flip` function.
+  //.
+  //. This function is very similar to [`flip`](#flip), except that its first
+  //. argument must be curried. This allows it to work with manually curried
+  //. functions.
+  //.
+  //. ```javascript
+  //. > S.C(R.concat, 'foo', 'bar')
+  //. 'barfoo'
+  //.
+  //. > R.filter(S.C(R.gt, 0), [-1, -2, 3, -4, 4, 2])
+  //. [3, 4, 2]
+  //. ```
+  S.C =
+  def('C',
+      {},
+      [$.Function, b, a, c],
+      function(f, x, y) { return f(y)(x); });
+
+  //# B :: (b -> c) -> (a -> b) -> a -> c
+  //.
+  //. The B combinator. Takes two functions and a value and returns the result
+  //. of applying the first function to the result of applying the second to the
+  //. value. Equivalent to Haskell's `(.)` function and [`compose`](#compose).
+  //.
+  //. ```javascript
+  //. > S.B(Math.sqrt, S.inc, 99)
+  //. 10
+  //. ```
+  S.B =
+  def('B',
+      {},
+      [$.Function, $.Function, a, c],
+      compose3);
+
+  //# S :: (a -> b -> c) -> (a -> b) -> a -> c
+  //.
+  //. The S combinator. Takes a curried binary function, a unary function and a
+  //. value and returns the result of applying the binary function to the value
```

Comment:
Are you sure about that first comma? School taught me a list is written like `a, b, c and d`. I wouldn't want to break off the last item from the list by placing that comma. I totally agree with the second comma though, because the rest of the sentence shouldn't be confused as being part of the list.

---

Some closer research shows that this rule differs between the UK and the US. The US *does* place that extra "[oxford comma](https://en.wikipedia.org/wiki/Serial_comma)". Since you're in the US, and the rest of the documentation is probably in en_US, I'll just add that extra comma. :)